sys-apps/ripgrep/ripgrep-0.10.0.recipe

```diff
@@ -0,0 +1,64 @@
+SUMMARY="Combines the usability of ag with the raw speed of grep"
+DESCRIPTION="ripgrep is a line-oriented search tool that recursively searches \
+your current directory for a regex pattern while respecting your gitignore \
+rules. It is similar to other popular search tools like The Silver Searcher, \
+ack and grep."
+HOMEPAGE="https://github.com/BurntSushi/ripgrep"
+COPYRIGHT="2015 Andrew Gallant"
+LICENSE="Unlicense
+	MIT"
+REVISION="1"
+SOURCE_URI="$HOMEPAGE/archive/$portVersion.tar.gz"
+CHECKSUM_SHA256="a2a6eb7d33d75e64613c158e1ae450899b437e37f1bfbd54f713b011cd8cc31e"
+SOURCE_FILENAME="ripgrep-$portVersion.tar.gz"
+
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+commandSuffix=$secondaryArchSuffix
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	commandSuffix=
+fi
+
+PROVIDES="
+	ripgrep$secondaryArchSuffix = $portVersion
+	cmd:rg$commandSuffix
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	lib:libxslt$secondaryArchSuffix
+	rust_bin$secondaryArchSuffix
+	asciidoc
```

Comment:
Suggestion:
```diff
-	rust_bin$secondaryArchSuffix
-	asciidoc
```
rust_bin will be fetched with the cmd:cargo$secondaryArchSuffix
put asciidoc also in BUILD_PREREQUIRES